src/imgui/FileListWidget.hh

```diff
@@ -29,6 +31,8 @@ public:
 	std::function<void()> drawAction; // default (only) calls drawTable()
 	std::function<void(const Entry&)> selectAction; // MUST be overwritten
 	std::function<void(const Entry&)> hoverAction; // default: nothing
+	std::function<void(const Entry&)> singleClickAction; // default: nothing
+	std::function<void(const Entry&)> doubleClickAction; // default: nothing
```

Comment:
See above, I think we only need 1 extra action, not 2.

Reply:
OK, fixed, thanks.